### PR TITLE
Cmake cleanup

### PR DESCRIPTION
**Pull request by manoj-freyr**
to remove code download dependency on yaml-cpp and gtest

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -241,82 +239,29 @@ message (STATUS "CPACK_GENERATOR ${CPACK_GENERATOR}" )
 
 ################################################################################
 # Download and unpack yaml-cpp at configure time
```
Comment (jkottiku) [inline CMakeLists.txt:241]: Update the comment.

File: CMakeLists.txt [hunk 2]
```diff
@@ -241,82 +239,29 @@ message (STATUS "CPACK_GENERATOR ${CPACK_GENERATOR}" )
...
+if (yaml-cpp_FOUND)
+  message("yaml-cpp found")
+else() 
+  message(FATAL_ERROR "yaml-cpp not found !!! Install  to proceed ...")
+endif(yaml-cpp_FOUND)
 
 ################################################################################
 ## GOOGLE TEST
 if(RVS_BUILD_TESTS)
   # Download and unpack googletest at configure time
```
Comment (jkottiku) [inline CMakeLists.txt:253]: Update the comment.

**Review by jkottiku — APPROVED** [review]
Update the comments. Rest looks good.

File: CMakeLists.txt (hunk 1 above)
Comment (manoj-freyr) [reply CMakeLists.txt]: done

File: CMakeLists.txt (hunk 2 above)
Comment (manoj-freyr) [reply CMakeLists.txt]: done

**Pull request merged**